share/completions/sv.fish

```diff
@@ -10,12 +10,19 @@ set -l commands \
     shutdown force-stop force-reload force-restart force-shutdown \
     try-restart check
 
+set -l search_svdirs \
+    /run/runit/runsvdir/current \
+    /run/runit/service \
+    /etc/services \
+    /services \
+    #
 
 function __fish_complete_sv_list_services
-    for dir in $SVDIR /run/runit/service/ /etc/runit/current /etc/runit/runsvdir/
+    for dir in $SVDIR $search_svdirs
         test -d $dir
         and break
     end
+    test -n $dir; or return
```

Comment:
Since $SVDIR is a user-defined variable, this needs to be quoted - `set -l SVDIR; test -n $SVDIR` *succeeds*.

Same for the above `test -d $dir`.

(another use case for #8265 - this entire thing would be `set -l dir (path filter -d -- $SVDIR $search_svdirs)[1]; or return`)